cnd/src/http_api/routes/index.rs

```diff
@@ -96,24 +96,45 @@ pub async fn post_herc20_halight_bitcoin(
         .map_err(warp::reject::custom)
 }
 
-// `warp::reply::Json` is used as a return type to please the compiler
-// until proper logic is implemented
 #[allow(clippy::needless_pass_by_value)]
 pub async fn post_halight_bitcoin_herc20(
     body: serde_json::Value,
-    _facade: Facade,
-) -> Result<warp::reply::Json, Rejection> {
-    let _body = Body::<HalightLightningBitcoin, Herc20EthereumErc20>::deserialize(&body)
+    facade: Facade,
+) -> Result<impl Reply, Rejection> {
+    let body = Body::<HalightLightningBitcoin, Herc20EthereumErc20>::deserialize(&body)
```

Comment:
Is there any reason you elected to _not_ shorten these struct identifiers?

Reply:
Agreed. I think we were just using what was there and weren't sure if we should modify them. We'll shorten them!